Makefile

```diff
@@ -32,6 +32,12 @@ endif
 PACKAGE_NAME=Linux-Entra-SSO
 
 RELEASE_TAG ?= $(shell git describe --match "v[0-9].[0-9]*" --dirty)
+ifdef VERSION
+PLUGIN_VERSION=$(VERSION)
+else
+PLUGIN_VERSION=$(shell jq -r '.version' platform/firefox/manifest.json)
+endif
+
 ARCHIVE_NAME=$(PACKAGE_NAME)-$(RELEASE_TAG)
```

Comment:
Why differentiating archive and plugin version? For devs installing from source, it will give them the wrong information about what they actually deployed.

Reply:
We need to differentiate, as we need to strip the "v" prefix. But you're right, we should better use the `RELEASE_TAG` as a base.